{"name": "test_ke", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'gp_login'", "trace": "driver_fixture = <appium.webdriver.webdriver.WebDriver (session=\"0e7b83d5-90bd-4243-ba0f-6351e0c81c1e\")>\n\n    @pytest.fixture(scope='session',autouse=False)\n    def gp_login_fixture(driver_fixture):\n        csv_file=BASE_DIR + '\\datas\\login.csv'\n        print(\"csv_file:%s\"%csv_file)\n        driver=driver_fixture\n        com=Common(driver)\n        data=com.get_csv_data(csv_file,2)\n        time.sleep(2)\n        trd=First(driver)\n        trd.to_login()\n        logn=Login(driver)\n>       logn.gp_login(data[0],data[1])\nE       AttributeError: 'Login' object has no attribute 'gp_login'\n\nconftest.py:54: AttributeError"}, "start": 1652795334188, "stop": 1652795334188, "uuid": "b7ab9bfe-7184-4b2b-a875-49241221b5fe", "historyId": "57109162688f3b246f3278f2d90a75a4", "testCaseId": "22b51dfb07cb1d562a233f588e9a3110", "fullName": "testGp.test_new_Gp.TestGp#test_ke", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_fixture')"}, {"name": "parentSuite", "value": "testGp"}, {"name": "suite", "value": "test_new_Gp"}, {"name": "subSuite", "value": "TestGp"}, {"name": "host", "value": "Win102021AVUCSJ"}, {"name": "thread", "value": "10404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testGp.test_new_Gp"}]}